schemas/oic.sec.cred.json

```diff
@@ -62,7 +62,7 @@
           }
         }
       },
-      "required": [ "credid", "subjectuuid", "credtype" ]
```

Comment:
OK. But we'll have to change the text in CR-1485 if so - since it express behavior for cases where the credid is omitted...